ios-sdkTests/MapzenSearchDataConverterTests.swift

```diff
@@ -0,0 +1,154 @@
+    XCTAssertEqual(MapzenSearchDataConverter.wrapLayerFilter(.country), .country)
+    XCTAssertEqual(MapzenSearchDataConverter.wrapLayerFilter(.region), .region)
+    XCTAssertEqual(MapzenSearchDataConverter.wrapLayerFilter(.locality), .locality)
+    XCTAssertEqual(MapzenSearchDataConverter.wrapLayerFilter(.localadmin), .localadmin)
+    XCTAssertEqual(MapzenSearchDataConverter.wrapLayerFilter(.neighbourhood), .neighbourhood)
+    XCTAssertEqual(MapzenSearchDataConverter.wrapLayerFilter(.coarse), .coarse)
+  }
+
+  func testWrapPoint() {
+    let point = GeoPoint(latitude: 70.0, longitude: 40.0)
+    let wrapped = MapzenSearchDataConverter.wrapPoint(point)
+    XCTAssertEqual(point, wrapped.point)
+  }
+
+  func testUnwrapPoint() {
+    let point = MzGeoPoint(latitude: 70.0, longitude: 40.0)
+    let unwrapped = MapzenSearchDataConverter.unwrapPoint(point)
+    XCTAssertEqual(point.point, unwrapped)
+  }
+
+  func testUnwrapQueryItems() {
+    let items = [MzPlaceQueryItem.init(placeId: "id", dataSource: .geoNames, layer: .address),
+                 MzPlaceQueryItem.init(placeId: "anotherid", dataSource: .quattroshapes, layer: .region)]
+    let unwrapped = MapzenSearchDataConverter.unwrapQueryItems(items)
+    XCTAssertEqual(unwrapped[0].placeId, "id")
+    XCTAssertEqual(unwrapped[0].dataSource, .GeoNames)
+    XCTAssertEqual(unwrapped[0].layer, .address)
+    XCTAssertEqual(unwrapped[1].placeId, "anotherid")
+    XCTAssertEqual(unwrapped[1].dataSource, .Quattroshapes)
+    XCTAssertEqual(unwrapped[1].layer, .region)
+    XCTAssertEqual(unwrapped.count, 2)
+  }
+
+  func testUnwrapQueryItem() {
+    let item = MzPlaceQueryItem.init(placeId: "id", dataSource: .openStreetMap, layer: .venue)
+    let unwrapped = MapzenSearchDataConverter.unwrapQueryItem(item)
+    XCTAssertEqual(unwrapped.placeId, "id")
+    XCTAssertEqual(unwrapped.dataSource, .OpenStreetMap)
+    XCTAssertEqual(unwrapped.layer, .venue)
+  }
+
+  func testWrapQueryItems() {
+    let items = [PeliasPlaceQueryItem.init(placeId: "id", dataSource: .GeoNames, layer: .address),
+                 PeliasPlaceQueryItem.init(placeId: "anotherid", dataSource: .Quattroshapes, layer: .region)]
+    let wrapped = MapzenSearchDataConverter.wrapQueryItems(items)
+    XCTAssertEqual(wrapped[0].placeId, "id")
+    XCTAssertEqual(wrapped[0].dataSource, .geoNames)
+    XCTAssertEqual(wrapped[0].layer, .address)
+    XCTAssertEqual(wrapped[1].placeId, "anotherid")
+    XCTAssertEqual(wrapped[1].dataSource, .quattroshapes)
+    XCTAssertEqual(wrapped[1].layer, .region)
+    XCTAssertEqual(wrapped.count, 2)
+  }
+
+  func testWrapQueryItem() {
+    let item = PeliasPlaceQueryItem.init(placeId: "id", dataSource: .OpenStreetMap, layer: .venue)
+    let wrapped = MapzenSearchDataConverter.wrapQueryItem(item)
+    XCTAssertEqual(wrapped.placeId, "id")
+    XCTAssertEqual(wrapped.dataSource, .openStreetMap)
+    XCTAssertEqual(wrapped.layer, .venue)
+  }
+
```

Comment:
I'm not seeing whitespace on this line:
<img width="710" alt="screen shot 2017-03-23 at 2 16 19 pm" src="https://cloud.githubusercontent.com/assets/494323/24268424/5a91e670-0fd3-11e7-84f1-dfc5b09c1c85.png">

Was it on a different line?